packages/engine-core/src/tasks/task.rnv.crypto.encrypt.ts

```diff
@@ -92,6 +95,8 @@ const _checkAndConfigureCrypto = async (c: RnvContext) => {
         cnf.crypto = {
             path: `./${location}/privateConfigs.enc`,
         };
+        if (cnf.crypto) c.files.project.config.crypto = cnf.crypto;
```

Comment:
@ElenaDiachenko the "if" statement is not necessary here considering you always create one line above